src/quo2/components/messages/system_message.cljs

```diff
@@ -0,0 +1,127 @@
+(ns quo2.components.messages.system-message
+  (:require [quo.react-native :as rn]
+            [quo.theme :as theme]
+            [quo2.components.buttons.button :as button]
+            [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.avatars.user-avatar :as user-avatar]
+            [quo2.components.avatars.icon-avatar :as icon-avatar]))
+
+(def themes
+  {:light {:text       colors/black
+           :icon       colors/primary-50
+           :time       colors/neutral-50
+           :background colors/neutral-5}
+   :dark  {:text       colors/white
+           :icon       colors/primary-50
+           :time       colors/neutral-40
+           :background colors/neutral-95}})
+
+(defn get-color [key]
+  (get-in themes [(theme/get-theme) key]))
+
+(defn sm-timestamp [timestamp-str]
+  [rn/view {:margin-left 6}
+   [text/text {:size  :label
+               :style {:color          (get-color :time)
+                       :text-transform :none}} timestamp-str]])
```

Comment:
Maybe put strings for all `text/text` components in a new line to improve readability. In this case I would put `timestamp-str` in a new line to separate it from other properties.